sdks/python/apache_beam/io/gcp/bigquery_read_it_test.py

```diff
@@ -172,6 +213,31 @@ def test_big_query_read_new_types(self):
           use_standard_sql=True)))
       assert_that(result, equal_to(expected_data))
 
+  def test_big_query_read_none_values(self):
```

Comment:
Looks like the test failed on Py3.

Reply:
Also, how about we incorporate None values to existing IT test scenarios?

Reply:
done